src/styles/blocks/contacts.scss

```diff
@@ -0,0 +1,35 @@
+.contacts {
+  &__title {
+    @extend %text-h2;
+    text-transform: uppercase;
+    margin-bottom: 10px;
+  
+    @include Desktop {
+      margin-bottom: 30px;
+    }
+  }
+
+  &__paragraph {
+    font-size: 14px;
+    margin-bottom: 30px;
+
+    @include Tablet {
+      font-size: 16px;
+      margin-bottom: 20px;
+    }
+
+    @include Desktop {
+      margin-bottom: 40px;
+    }
+  }
+
+  &__image {
+    @extend %bacground-images-settings;
+    background-image: url(../images/png/6.1_tablet.png);
```

Comment:
Use a more descriptive name for the image file, like 'eco-cosmetics-contacts-tablet.png' instead of '6.1_tablet.png'.